spark/v3.1/spark/src/main/java/org/apache/iceberg/spark/SparkWriteOptions.java

```diff
@@ -51,5 +51,8 @@ private SparkWriteOptions() {}
   public static final String HANDLE_TIMESTAMP_WITHOUT_TIMEZONE =
       "handle-timestamp-without-timezone";
 
+  // Output partition spec ID where writes should go.
```

Comment:
Nit: comment seems to just repeat information of the field name, doesnt seem worth it as is.  Maybe remove, or improve it, maybe 'partition spec to apply on newly written files'

Reply:
Agreed. Removed it as it is self-explanatory.